plugins/jobtype/src/azkaban/jobtype/HadoopHiveJob.java

```diff
@@ -22,12 +22,15 @@
 import java.io.IOException;
 import java.lang.reflect.InvocationTargetException;
 import java.lang.reflect.Method;
```

Comment:
Can we handle the case of killing hadoop jobs if someone kills job process in azkaban executor box ?